portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,12 +33,26 @@
 public class DataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String commentcount = request.getParameter("count");
+    int maxcomments;
+    try {
+        maxcomments = Integer.parseInt(commentcount);
+    } catch(NumberFormatException e){
+        // Display nothing for now.
```

Comment:
If it's not parseable shouldn't you just throw the error?  I'm thinking it'll be something like:

int maxComments = request.getParameter("count") == null ? 0 : Integer.parseInt( request.getParameter("count"));

Reply:
Changed to your suggestion.